garnett/shapes.py

```diff
@@ -191,6 +191,19 @@ def pos_string(self):
 
         return shape_def
 
+    @property
+    @_json_sanitize
+    def type_shape(self):
+        """Shape as dictionary. Example:
+
+           >>> SphereUnionShape([0.5, 0.5, 0.5], [[0, 0, 1.0], [0, 1.0, 0], [1.0, 0, 0]]).type_shape
+           {'type': 'SphereUnion', 'diameters': [0.5, 0.5, 0.5],
+            'centers': [[0, 0, 1.0], [0, 1.0, 0], [1.0, 0, 0]]}
+        """
+        return {'type': 'SphereUnion',
```

Comment:
The order of the dictionary keys here should match the example since dictionaries are ordered in Python >=3.7. This should be the same order as the constructor.

Reply:
Done